TestResultSummaryService/routes/getParserProps.js

```diff
@@ -1,4 +1,10 @@
 const benchmarkMetric = require( '../parsers/BenchmarkMetric');
 module.exports = async ( req, res ) => {
-    res.json({benchmarkMetric});
+    let { benchmarkName } = req.query;
+    for(let key of Object.keys(benchmarkMetric)) {
+        if(benchmarkName.includes(key)){
+            res.json(benchmarkMetric[key]["metrics"]);
```

Comment:
We should just return when we find the benchmark parser key instead of continuously looping through all keys.

Reply:
The above code sends the response when there is a match. It does not continuously loop through all keys. Please read the express doc.

Reply:
Sorry, you're right! I overlooked that.